### PR TITLE
Add Pangolin<>PangolinParachain related consts

### PR DESCRIPTION
**Pull request by furoxr**

### CODE REVIEW

File: primitives/chain-pangolin/src/lib.rs [hunk 1]
```diff
@@ -62,6 +62,9 @@ pub const BEST_FINALIZED_PANGOLIN_HEADER_METHOD: &str = "PangolinFinalityApi_bes
 /// Name of the `ToPangolinOutboundLaneApi::message_details` runtime method.
 pub const TO_PANGOLIN_MESSAGE_DETAILS_METHOD: &str = "ToPangolinOutboundLaneApi_message_details";
 
+/// Name of the With-PangolinParachain message pallet instance that is deployed at bridged chains.
+pub const WITH_PANGOLIN_PARACHAIN_MESSAGES_PALLET_NAME: &str = "BridgePangolinParachainMessages";
```
Comment (AurevoirXavier) [inline primitives/chain-pangolin/src/lib.rs:66]: This is not belong to Pangolin.

Comment (AurevoirXavier) [reply primitives/chain-pangolin/src/lib.rs:66]: https://github.com/darwinia-network/darwinia-bridges-substrate/blob/main/primitives/chain-pangolin-parachain/src/lib.rs#L55

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]
-

File: primitives/chain-pangolin/src/lib.rs (hunk 1 above)
Comment (furoxr) [reply primitives/chain-pangolin/src/lib.rs:66]: In this branch: https://github.com/darwinia-network/darwinia-common/blob/c6c88b1759d345614301882156e5568d83d853cc/node/runtime/pangolin/src/lib.rs#L265

Comment (AurevoirXavier) [reply primitives/chain-pangolin/src/lib.rs:66]: You should modify `primitives/chain-pangolin-parachain`.

File: primitives/runtime/src/lib.rs [hunk 2]
```diff
@@ -60,6 +60,12 @@ pub const ROCOCO_CHAIN_ID: ChainId = *b"roco";
 /// Bridge-with-Wococo instance id.
 pub const WOCOCO_CHAIN_ID: ChainId = *b"woco";
 
+/// Bridge-with-Pangolin instance id.
+pub const PANGOLIN_CHAIN_ID: ChainId = *b"pagl";
+
+/// Bridge-with-PangolinParachain instance id.
+pub const PANGOLIN_PARACHAIN_CHAIN_ID: ChainId = *b"ppal";
```
Comment (AurevoirXavier) [inline primitives/runtime/src/lib.rs:67]: What if there is a Pangoro Parachain?

Comment (AurevoirXavier) [reply primitives/runtime/src/lib.rs:67]: Suggest

`pglp` pangolin parachain
`pgrp` pangoro parachain

**Pull request merged**